src/App.tsx

```diff
@@ -1,9 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { Movie } from './types/Movie';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const filterFilms = (element: string, query: string): boolean => {
+  return element.toLowerCase().includes(query.toLowerCase());
+};
+
+const getFilteredFilms = (
+  films: Movie[],
+  query: string,
+) => {
+  return films.filter(film => {
+    const filterByTitle = filterFilms(film.title, query);
+    const filterByDescription = filterFilms(film.description, query);
+
+    return filterByTitle || filterByDescription;
+  });
+};
```

Comment:
Try this syntax. You have 2 extra variables.
```suggestion
const getFilteredFilms = (
  films: Movie[],
  query: string,
) => films.filter(film => (
  filterFilms(film.title, query) || filterFilms(film.description, query)
));
```